api/src/shipit_api/common/config.py

```diff
@@ -459,10 +459,11 @@
         {"name": "push_thunderbird", "in_previous_graph_ids": True},
         {"name": "ship_thunderbird", "in_previous_graph_ids": True},
     ],
+    "android-components": [{"name": "ship_android_components", "in_previous_graph_ids": True}],
```

Comment:
I'll let @JohanLorenzo weigh in on the phase name, but if [this holds true](https://github.com/mozilla-mobile/android-components/pull/9132#pullrequestreview-545276915) and Sarah copies in the Fenix block, then the phase will be named `ship`. This doesn't conflict with Fenix's `ship` because they're pointing at different repos. (Devedition and Firefox are built from the same repo, so they need unique names to differentiate.)

Reply:
Yup, I think `ship` is better suited here for the reason Aki explained :) 

```suggestion
    "android_components": [{"name": "ship", "in_previous_graph_ids": True}],
```